src/assets/stylesheets/extensions/_codehilite.scss

```diff
@@ -216,15 +216,23 @@ $codehilite-whitespace: transparent;
   // If code blocks are wrapped with codehilite, the styles must be adjusted
   // so the marker stretches to the whole width and the padding is respected
   .codehilite {
+    position: relative;
     margin: 1em 0;
-    padding: 1rem 1.2rem 0.8rem;
+    padding: 0;
     border-radius: 0.2rem;
     background-color: $md-code-background;
     color: $md-code-color;
     line-height: 1.4;
-    overflow: auto;
```

Comment:
Understood, I believe this should check out okay.  In this case we are just flipping the the attributes with the wrapper and code element.  I'm actually using this over in my current pymdown-extensions docs.  But maybe their is a corner case I'm not aware of.